tests/local_providers/allocine_stocks_test.py

```diff
@@ -1714,6 +1714,19 @@ def test_should_return_the_right_stock_number(self):
         assert stock_number == 12
 
 
+class FormatNaiveDateToUtcTest:
+    def test_should_convert_date_to_utc_timezone(self):
+        # Given
+        naive_local_date = datetime(2019, 12, 3, 20, 0)
```

Comment:
si tu renommes la fonction comme proposé ci dessus :
```suggestion
        local_date = datetime(2019, 12, 3, 20, 0)
```